Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pytest = "*"
+pylint = "*"
```

Comment:
You might want to pin these to specific versions—tends to make updating dependencies a little more controlled (at least that's been my experience especially in Ruby with bundler; pip might have a better approach to this)

Reply:
I tried to do this, but it kept getting errors. I think I'm probably not understanding the syntax or something.